_rules/inline-link-in-paragraph-has-distinguishable-style-on-hover-and-focus-548868.md

```diff
@@ -19,22 +19,25 @@ acknowledgments:
 
 This rule applies to any [visible][] HTML element that is a [semantic link][], for which all the following is true:
 
-- **link text**: The element has [visible][] [text nodes][text node] as [descendants][descendant] in the [flat tree][]; and
-- **non-link line text**: The element is [rendered on a line][] containing [visible][] [text nodes][text node] that are not [descendants][descendant] in the [flat tree][] of a [semantic link][].
+- **link text**: the element has [visible][] [text nodes][text node] as [descendants][descendant] in the [flat tree][]; and
+- **non-link line text**: the element is [rendered on a line][] containing [visible][] [text nodes][text node] that are not [descendants][descendant] in the [flat tree][] of a [semantic link][]; or
+- **different color**: the element's [foreground color][] and the [foreground color][] of the **non-link line text** elements have a [highest possible contrast][] that is at least 3.0:1, or the element's [background color][] and the [background color][] of the **non-link line text** elements have a [highest possible contrast][] that is at least 3.0:1.
 
 ## Expectation
 
-For each test target all the following is true:
+For each test target, at least one of the following is true when it is [focused][] or [hovered][]:
 
-- a [highest possible contrast][] that is at least 3.0:1 between its [foreground color][] and the [foreground color][] of the [visible][] [text nodes][text node] [rendered on the same line][rendered on a line] that are not [descendants][descendant] in the [flat tree][] of a [semantic link][]; and
-- a [distinguishing style][] both when the target element [gains focus][focused] and the target element is [hovered][].
+- **distinguishing style**: the element has a [distinguishable style][] not based on color alone from the **non-link line text** elements; or
+- **distinguishing content**: the element has content (such as an image or text) located in the proximity of the element (i.e. there is no other content visually in between this content and the target element), that indicates the test target is a link.
```

Comment:
I think something can be done here to clarify this assumption.

* we say "proximity" (which to me does not convey the idea of adjacent or side by side)
* and then we say "there is no other content visually between this content and the test target" (which seems to convey the idea that such content needs to be adjacent to the test target

which seems awkward to me.

Maybe:

```suggestion
- **distinguishing content**: the element has adjacent content (such as an image or text) that indicates the test target is a link.
```